packages/taco/src/conditions/context/context.ts

```diff
@@ -178,6 +179,26 @@ export class ConditionContext {
         }
       }
     }
+    // If it's a JSON API condition, check url and query
+    if (condition.conditionType === JsonApiConditionType) {
+      const urlComponents = condition.endpoint.replace("https://", "").split("/");
+      for (const param of urlComponents ?? []) {
+        if (this.isContextParameter(param)) {
+          requestedParameters.add(param);
+        }
+      }
+      if (condition.query) {
+        const queryParams = condition.query.match(":[a-zA-Z_]*");
```

Comment:
`*` will match zero tokens but we can use `+`

```suggestion
        const queryParams = condition.query.match(":[a-zA-Z_]+");
```